playbooks/openshift-web-console/private/redeploy-certificates.yml

```diff
@@ -1,35 +1,48 @@
 ---
+
 - name: Update web console certificates
   hosts: oo_first_master
   vars:
   roles:
   - lib_openshift
   - openshift_facts
   tasks:
-  - name: Remove certificates secret
-    oc_obj:
-      name: webconsole-serving-cert
-      kind: secret
-      state: absent
-      namespace: openshift-web-console
+  - name: Remove certificates secret in openshift-web-console
+      shell: oc delete secret console-serving-cert -n openshift-console
```

Comment:
`oc_obj` would translate this to oc calls, why not use the module?

Reply:
the ob_obj doesn't absent the secret, and doesn't roll out new pods on clusters upgraded from previous versions.